src/gentropy/window_based_clumping.py

```diff
@@ -13,17 +13,21 @@ def __init__(
         session: Session,
         summary_statistics_input_path: str,
         study_locus_output_path: str,
+        distance: int = 500_000,
+        collect_locus: bool = False,
+        collect_locus_distance: int = 500_000,
```

Comment:
So it seems this clumping step is flexible to do or not do locus collection.

Reply:
Without having more pieces of the puzzle, I struggle to imagine whether we would run the clumping only once or twice (with and without collection).